screens/WalletNavigator/screens/WalletOnboarding.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+import { View, Text, TouchableOpacity, ScrollView } from 'react-native'
+import tailwind from 'tailwind-rn'
+import { Ionicons } from '@expo/vector-icons'
+import { translate } from '../../../translations'
+import { PrimaryColor } from '../../../constants/Theme'
+import { useNavigation } from '@react-navigation/native'
+import { useWalletAPI } from '../../../hooks/wallet/WalletAPI'
+import { useDispatch } from 'react-redux'
+
+export function WalletOnboarding (): JSX.Element {
+  const WalletAPI = useWalletAPI()
+  const dispatch = useDispatch()
+  const navigator = useNavigation()
+
+  function onDebugPress (): void {
+    // TODO(fuxingloh): this should only be available in debug mode
+    WalletAPI.setMnemonicAbandon23(dispatch)
+  }
+
+  return (
+    <ScrollView style={tailwind('flex-1 py-8 bg-gray-100')} testID='wallet_onboarding'>
+      <View style={tailwind('flex items-center')}>
+        <TouchableOpacity delayLongPress={5000} onLongPress={onDebugPress}>
+          <View style={tailwind('flex bg-white justify-center items-center rounded-full h-16 w-16')}>
+            <Ionicons size={24} name='wallet' color='#999' />
+          </View>
+        </TouchableOpacity>
+
+        <Text style={tailwind('font-bold mt-4 text-gray-600')}>
+          {translate('screens/WalletOnboarding', 'No wallets')}
+        </Text>
+      </View>
+
+      <View style={tailwind('mt-8')}>
```

Comment:
I think we're missing some screens here.
1. Empty Wallet Screen
2. Wallet Display screen
3. Add/Create wallet screen

Currently the `WalletOnboarding` screen combines 1 and 3, I'm not sure if this is agreed upon by design team

Reply:
Will be tackled in another PR with explanation.